openfisca_paris/paris.py

```diff
@@ -226,8 +226,15 @@ def formula(famille, period, legislation):
         aspa_reference = famille('paris_aspa_reference', period)
         ressources = max_(aspa_reference, base_ressources)
 
-        return (loyer + charges_forfaitaires_logement - aide_logement) / ressources
+        # Utiliser si les ressources sont à 0
+        ressources_inf = fl_(ressources, inf)
 
+        return div_(
+            (loyer + charges_forfaitaires_logement - aide_logement),
+            ressources,
+            out=ressources_inf,
+            where=ressources != 0
+        )
```

Comment:
Revue naïve: avec cette modification, le taux d'effort vaut l'infini si les ressources sont nulles. Est-ce que c'est bien ce qu'on veut ? Est-ce que tu pourrais ajouter un scénario de test pour appliquer ce cas ?

Reply:
De mémoire, c'est déjà visible en mode : Tu lances les tests sans le fix et tu les relances avec le fix @Shamzic

Reply:
CE que je veux dire, c'est que si `paris_taux_effort` vaut  l'nfini, alors `paris_condition_taux_effort` vaut `true` et je suis pas si sûr que ça soit forcément ce résultat qui est attendu, mais peut-être que c'est ok après tout

Reply:
Pour moi, si tu n'as pas de revenu ton taux d'effort est infini si tu dois par exemple payer un loyer.
Si tu as un peu de revenu mathématiquement on peux définir ce taux d'effort comme pourrait le faire une banque dans le cas d'une demande de prêt. Donc pour moi c'est ok. Sauf si tu as une idée pour une autre implémentation. De mon côté, je te propose de partir dans cette direction et itérer. Cela permettra de supprimer l'erreur dans les logs.

Reply:
Ok, du coup je mets en image le test qui vérifie la modification actuelle dans le fichier tests/unittests/paris.yaml :

```yaml
- name: Condition vérifiant si le taux d'effort est suffisamment élevé
  period: 2019-02
  input:
    paris_taux_effort: [0.3, 0.299, infinity]
  output:
    paris_condition_taux_effort: [true, false, true]
```